lib/main_common.pm

```diff
@@ -1492,6 +1492,7 @@ sub load_extra_tests_desktop {
         }
         loadtest "console/check_default_network_manager";
     }
+    loadtest "x11/rrdtool_x11" ;
```

Comment:
@martinsmarcelo, iirc the rrdtool tests are working just for gnome atm, so I would suggest to move them to inside the if at [L1483](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/7451/files#diff-d3e0a6c7df772cff7741832a6250d5afR1483) and move the comments about the network part a little bit under, so that rrdtool test is scheduled before the network tests, if DESKTOP=gnome I should have looked closer to where the load was before :)

Reply:
ok, done!